src/ansys/tools/path/path.py

```diff
@@ -1155,17 +1147,18 @@ def version_from_path(product: PRODUCT_TYPE, path: str) -> int:
         Integer version number (for example, 231).
 
     """
+    product_name = PRODUCT_EXE_INFO[product]["name"]
     if not isinstance(path, str):
         raise ValueError(
             f'The provided path, "{path}", is not a valid string. '
-            f'If "{product}" is not installed in the default location, use ``save-ansys-path`` '
-            f"to save the path so it can be found by ``ansys-tools-path``."
+            f"Run the following command to save the path to the {product_name} executable:\n\n"
+            f"    save-ansys-path --name {product} /path/to/{product}-executable\n"
```

Comment:
Not sure we need those spaces.
```suggestion
            f"save-ansys-path --name {product} /path/to/{product}-executable\n"
```